### DISCUSSION TITLE
Trying to pull  openthread/environment:latest getting error

### DISCUSSION THREAD

**Discussion by kishanharale** [openthread/openthread#10562] [Q&A]
Trying to  pull openthread/environment:latest in docker on Mac but show this error
<img width="1253" alt="Screenshot 2024-07-30 at 9 51 51 PM" src="https://github.com/user-attachments/assets/f7537a45-743b-45dc-b9a0-85be57c2ad7b">
I am clueless I did research on this but did not got solution can anyone help me out.

**Comment by jwhui** [comment]
The current `buildx` platform list only includes `linux/amd64`:
https://github.com/openthread/openthread/blob/509596fe2c96f8ccec08adc130516fa6cc13a1da/.github/workflows/docker.yml#L71

I'll see if it's an easy fix to just add `linux/arm64`.
